base/libgit2/diff.jl

```diff
@@ -36,9 +36,12 @@ function Base.count(diff::GitDiff)
 end
 
 function Base.getindex(diff::GitDiff, i::Csize_t)
+    if i < 1 || i >= count(diff)
```

Comment:
is it a bounds error to ask for `diff[count(diff)]` or is that the last element?